ansible/roles/ml-analytics-service/defaults/main.yml

```diff
@@ -42,7 +42,7 @@ ml_analytics_azure_account_name: "{{ml_analytics_azure_account_name}}"
 ml_analytics_azure_container_name: "{{ml_analytics_azure_container_name}}"
 ml_analytics_observation_azure_blob_path: "observation/status/"
 ml_analytics_project_azure_blob_path: "reports/"
-ml_analytics_redis_host: "{{groups['ml-redis'][]}}"
+ml_analytics_redis_host: "{{ml_redis_url | default(groups['dp-redis'][0])}}"
```

Comment:
its not redis url right? it's `ml_redis_host`

Reply:
url is something which has protocol://endpoint. It's just ip address of the service.

Reply:
Ok got it. Changing now.